Add access to h2o model from API

pmbroda opened the pull request:
You can run this through Rscript.exe

kinbiko commented on r/model_production/h2oserv.R:
```diff
@@ -0,0 +1,33 @@
+                         snow = as.logical(is_snow),
+                         fog = as.logical(is_fog))
+  
+  print(df_check)
```
remove debugs

kinbiko commented on r/model_production/h2oserv.R:
```diff
@@ -0,0 +1,33 @@
+  print(df_check)
+  df_check_id <- h2o.getId(df_check)
+  
+  rfmodel <- h2o.getModel("DRF_model_R_1520781866877_1")
```
Let's create a name of the model that's consistent instead of relying on the (unix) timestamp

pmbroda commented:
I have cleaned up all the files and added comments. I am having issues stopping the server short of killing it in non-interactive mode. In production that should not really be a problem - more of a testing headache.

kinbiko commented:
Sweet! Looks like you even fixed the paths!
